cinema/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.CharField(max_length=255)
```

Comment:
Consider using `TextField` for `description` if you expect the text to be longer than 255 characters. Otherwise, using `CharField` with `max_length` is appropriate.